docs/source/model_design/general_reference_material/bliss/index.rst

```diff
@@ -456,9 +456,29 @@ In the NHANES example, the full first sentence of the above quote was:
 3.2.1 Global Burden of Disease (GBD)
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-.. todo::
-  Describe the best methodology/terminology for working with the "sex" variable from GBD
+The GBD study is very complex and many different data sources. Here, 
+we provide a general overview of how the demographics team works with the "sex" 
+variable and some of the limitations. To understand "sex" in the context of a 
+particular disease or risk factor, please reference the `GBD methods appendix <https://www.thelancet.com/journals/lancet/article/PIIS0140-6736(20)30925-9/fulltext#supplementaryMaterial>`_ to 
+see the exact data sources used. 
+
+The high level steps used for calculating population estimates for each 
+country, year, age group and sex are: 
+
+  #. First, GBD uses fertility rate estimates from hospital records and surveys such as DHS to define the population. A standard sex ratio at birth is applied to all. 
```

Comment:
I'm so curious where this ratio comes from!

Reply:
This is the best I got "sex ratio of observed prevalence in countries and territories with nationally representative surveys." 

However, there are some interesting things I don't get into here! Basically it is assumed that this ratio is the same for everyone (its about 1.05 males to 1 females) BUT in some countries where men and more desirable than women there might be terminated pregnancies based on sex. 

"Since the introduction of ultrasound technologies and the ability to conduct sex-selective abortions,
previously stable SRBs have shifted in some locations as a result of systematic sex preferences for children. SRBs in
recent years are particularly skewed in the Caucasus, south Asia, and east Asia."

Reply:
Huh interesting. Doesn't seem super conclusive about what kind of data source they used, but I would *assume* it is something reflecting sex assigned at birth. I think we can leave this as-is for now.